readme.md

```diff
@@ -30,8 +30,8 @@ Make all the changes smooth on hover (during 300ms):
 ## Checklist
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_catalog/report/html_report/)
+- [DEMO LINK](https://Q1W2E3R4T5Y6U7I8a.github.io/layout_catalog/)
+- [TEST REPORT LINK](https://Q1W2E3R4T5Y6U7I8a.github.io/layout_catalog/report/html_report/)
```

Comment:
Add these links to the PR description 
<img width="972" alt="image" src="https://github.com/user-attachments/assets/a39eea03-8cb4-4614-90f0-da7df5292b1d">